### ISSUE TITLE
Stylelint not finding .stylelintrc file

### ISSUE THREAD

**Issue by fernandofleury** [stylelint/stylelint#588]
![image](https://cloud.githubusercontent.com/assets/2780941/12037925/cdbd2a16-ae3a-11e5-951b-10ed3cf229c3.png)

![image](https://cloud.githubusercontent.com/assets/2780941/12037933/e033e5e0-ae3a-11e5-9e39-f44d348483c6.png)

```javascript
'use strict';
import gulp from 'gulp';
import postcss from 'gulp-postcss';
import rename from 'gulp-rename';

import paths from './paths';

import external from 'postcss-import';
import vars from 'postcss-simple-vars';
import extend from 'postcss-extend';
import lost from 'lost';
import color from 'postcss-color-function';
import media from 'postcss-custom-media';
import nested from 'postcss-nested';
import stylelint from 'stylelint';
import reporter from 'postcss-reporter';
import cssnano from 'cssnano';

export default () => {
  let processors = [
    external,
    vars,
    extend,
    lost,
    color,
    media,
    nested,
    stylelint,
    reporter({clearMessages: true}),
    cssnano
  ];

  return gulp.src(paths.src.css.glob)
    .pipe(postcss(processors))
    .pipe(rename('garden.min.css'))
    .pipe(gulp.dest(paths.src.css.dest));
};
```

If i provide manually with `configFile: '.stylelintrc'` I get a "double" log of issues:

```
src/css/atoms/typography.css
1:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
1:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
1:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
2:21	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
3:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
6:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
6:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
6:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
7:22	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
8:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
11:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
11:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
11:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
12:20	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
13:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
16:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
16:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
16:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
17:22	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
18:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
21:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
21:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
21:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
22:21	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
23:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
26:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
26:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
26:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
27:18	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]


src/css/atoms/typography.css
1:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
1:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
1:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
2:21	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
3:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
6:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
6:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
6:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
7:22	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
8:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
11:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
11:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
11:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
12:20	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
13:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
16:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
16:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
16:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
17:22	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
18:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
21:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
21:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
21:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
22:21	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
23:2	⚠  Unexpected whitespace after "}" (block-closing-brace-space-after) [stylelint]
26:3	⚠  Expected newline after "," (selector-list-comma-newline-after) [stylelint]
26:3	⚠  Unexpected whitespace after "," (selector-list-comma-space-after) [stylelint]
26:17	⚠  Unexpected whitespace after "{" (block-opening-brace-space-after) [stylelint]
27:18	⚠  Unexpected whitespace before "}" (block-closing-brace-space-before) [stylelint]
```

**Comment by fernandofleury** [comment]
This is my `.stylelintrc`

```json
{
  "rules": {
    "string-quotes": "double",
    "color-hex-case": "lower",
    "color-hex-length": "long",
    "color-no-hex": true,
    "number-leading-zero": "never",
    "number-zero-length-no-unit": true,
    "number-no-trailing-zeros": true,
    "function-calc-no-unspaced-operator": true,
    "function-comma-space-after": "always",
    "function-parentheses-newline-inside": "never-multi-line",
    "function-url-quotes": "double",
    "value-no-vendor-prefix": true,
    "value-list-comma-newline-after": "never-multi-line",
    "value-list-comma-newline-before": "never-multi-line",
    "value-list-comma-space-after": "always",
    "value-list-comma-space-before": "never",
    "unit-blacklist": ["px"],
    "declaration-no-important": true,
    "declaration-colon-space-after": "always",
    "declaration-colon-space-before": "never",
    "declaration-block-no-single-line": true,
    "declaration-block-semicolon-newline-after": "always",
    "declaration-block-semicolon-newline-before": "never-multi-line",
    "declaration-block-single-line-max-declarations": 1,
    "block-closing-brace-newline-after": "always",
    "block-closing-brace-newline-before": "always",
    "block-closing-brace-space-after": "never",
    "block-closing-brace-space-before": "never",
    "block-no-empty": true,
    "block-opening-brace-newline-after": "always",
    "block-opening-brace-space-after": "never",
    "block-opening-brace-space-before": "always",
    "selector-combinator-space-after": "always",
    "selector-combinator-space-before": "always",
    "selector-no-id": true,
    "selector-no-vendor-prefix": true,
    "selector-pseudo-element-colon-notation": "double",
    "selector-list-comma-newline-after": "always",
    "selector-list-comma-space-after": "never",
    "selector-list-comma-space-before": "never",
    "rule-nested-empty-line-before": "always",
    "rule-no-duplicate-properties": true,
    "rule-no-shorthand-property-overrides": true,
    "rule-non-nested-empty-line-before": "always",
    "rule-properties-order": "alphabetical",
    "rule-trailing-semicolon": "always",
    "media-feature-colon-space-after": "always",
    "media-feature-colon-space-before": "never",
    "media-feature-range-operator-space-after": "always",
    "media-feature-range-operator-space-before": "always",
    "media-query-parentheses-space-inside": "never",
    "media-query-list-comma-space-after": "always",
    "media-query-list-comma-space-before": "never",
    "comment-whitespace-inside": "always",
    "indentation": 2,
    "max-empty-lines": 2,
    "no-eol-whitespace": true,
    "no-missing-eof-newline": true
  }
}
```

**Comment by fernandofleury** [comment]
I've managed to make it work performing a cleanup of my rules. Thou not sure which was the responsible for it.
